web-app/src/components/tournament/creator/teamBuilder.vue

```diff
@@ -0,0 +1,234 @@
+<template>
+  <q-form @submit.prevent="submitAddTeam">
+    <q-card :style="$q.screen.lt.sm ? '' : 'height:75vh'">
+      <q-card-section class="q-ma-none q-pb-none">
+        <div class="text-h6">{{ $t('tournament.team.builderTitle') }}</div>
+
+        <q-input
+          clearable
+          clear-icon="close"
+          outlined
+          v-model="teamName"
+          :label="$t('tournament.team.name')"
+          :error-message="teamNameErrorMessage"
+          :error="isErrorInputTeamName"
+        />
+      </q-card-section>
+      <q-card-section class="q-ma-none q-py-none">
+        <q-virtual-scroll :items="players" separator style="max-height: 40vh">
+          <template v-slot="{ item }">
+            <q-item dense>
+              <q-item-section>
+                <q-item-label class="row">
+                  <q-item class="q-pa-none col-11">
+                    <q-item-section avatar>
+                      <q-avatar>
+                        <img :src="item.avatarSrc" />
+                      </q-avatar>
+                    </q-item-section>
+                    <q-item-section>
+                      {{ item.firstName }} {{ item.lastName }}
+                    </q-item-section>
+                  </q-item>
+
+                  <q-btn
+                    class="col-1"
+                    dense
+                    flat
+                    text-color="red"
+                    icon="close"
+                    @click="deletePlayer(item)"
+                  />
+                </q-item-label>
+              </q-item-section>
+            </q-item>
+          </template>
+        </q-virtual-scroll>
+      </q-card-section>
+      <q-card-section class="row items-stretch content-center q-pb-none">
+        <q-select
+          :class="$q.screen.gt.xs ? 'col-11 q-pb-none' : 'col-12'"
```

Comment:
```suggestion
          class="col-xs-11 col-12 q-pb-xs-none"
```
If I remember correctly, it will work